charts/datasaur/templates/start/elasticsearch-eck.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.elasticsearch.enabled }}
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: {{ .Release.Name }}-elasticsearch
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+spec:
+  version: 8.10.4
```

Comment:
is this what you did to override the helm chart and ES version?

Reply:
no.. ini buat yang self-hosted aku udah pake ECK, jdi ini udah latest and recommended approach from elastic @tonoman3g